app/services/typicode.js

```diff
@@ -25,13 +33,34 @@ exports.getAlbumById = albumId => {
       return requestAlbumPhotos(albumId);
     })
     .then(photos => {
-      album.photos = photos.map(photo => ({
-        url: photo.url,
-        thumbnailUrl: photo.thumbnailUrl
-      }));
+      mapPhotos(album, photos);
       return album;
     })
     .catch(error => {
       throw apiError(error.message);
     });
 };
+
+exports.getAlbums = (offset, limit, orderedBy) => {
+  logger.info('Requesting albums and photos');
+  let albums = {};
+  let photos = {};
+  return request(options(`${typicodePath}/photos`))
```

Comment:
I think that is better:
1- get all albums
2- slice the result
3- Get photos for each album 
4- Order the final result

Reply:
I agree

Reply:
You are right. Thanks!